VERSION

```diff
@@ -84,15 +84,15 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=30:2:10
+libmpi_so_version=30:3:10
```

Comment:
I did not see any changes to libmpi that were not in components since v2.1.2.  Did I miss something?

Reply:
oh okay, i didn't realize that's how you were looking at the deltas.

Reply:
according to this line of thinking, I'm reverting the r update to libopen_rte as well since there were only changes in mca components.